traffic_stops/routers.py

```diff
@@ -31,24 +31,26 @@ def db_for_write(self, model, **hints):
         logger.debug('db_for_write({}): {}'.format(state_db, name))
         return name
 
-    def allow_migrate(self, db, model):
+    def allow_migrate(self, db, app_label, model_name=None, **hints):
         # scenarios:
         #
         # default           traffic_stops_nc    False
         # default           traffic_stops_admin True
         # traffic_stops_nc  traffic_stops_admin False
         # traffic_stops_nc  traffic_stops_nc    True
         #
-        name = self._db_name(model)
-        if db == 'default':
-            if name in settings.DATABASES:
+        db_state = db[-2:]
+        app_is_state = app_label in ('nc', 'md')
+        if db_state == app_label:
+            ret = True
+        elif db == 'default':
+            if app_is_state:
                 ret = False
             else:
                 ret = True
         else:
-            if name in settings.DATABASES and db == name:
-                ret = True
-            else:
-                ret = False
-        logger.debug('allow_syncdb({}, {}): {}'.format(db, model, ret))
+            ret = False
+        logger.debug(
+            'allow_syncdb({}, {}): {} {} {}'.format(db, app_label, model_name,
+                                                    app_label, app_is_state, ret))
```

Comment:
5 ``{}`` in the format string, 6 parameters; did you intend to log something else?  IMO it should be enough to log the input and the output here, but either way is ok